controllers/storagerequest/storagerequest_controller.go

```diff
@@ -186,6 +188,7 @@ func (r *StorageRequestReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Watches(&rookCephv1.CephClient{}, enqueueForOwner).
 		Watches(&storagev1.StorageClass{}, enqueueStorageConsumerRequest).
 		Watches(&snapapi.VolumeSnapshotClass{}, enqueueStorageConsumerRequest).
+		Watches(&groupsnapapi.VolumeGroupSnapshotClass{}, enqueueStorageConsumerRequest).
```

Comment:
i think we dont need snapshot,groupsnapshot,pv,storageclass permission in storagerequest_controller. @leelavg can you please confirm it?

Reply:
hmm, I didn't get the GH notification 🤔. Anyways, yes, we don't watch for these resources and need the struct to be encoded in gRPC response.

Just rechecked existing code and I don't think anything necessary for provider mode. do we need more info than below inaddition to changes to storageclaimcontroller in https://github.com/red-hat-storage/ocs-client-operator/pull/168/files?
https://github.com/red-hat-storage/ocs-operator/blob/1ea3ed4d313d8f75742be8d05eebd93000e0b6f5/services/provider/server/server.go#L730-L735